resources/js/src/app/components/itemList/filter/ItemDropdown.spec.ts

```diff
@@ -0,0 +1,155 @@
+
+    let store: Store<State>;
+
+    beforeEach(() => {
+        store = new Vuex.Store({});
+    });
+
+    it('does not show a dropdown if the fixed value count is greater than the available filter values', () => {
+        const facet: Facet = {
+            cssClass: '',
+            findologicFilterType: 'select',
+            id: 'test',
+            isMain: false,
+            itemCount: 3,
+            name: 'Test',
+            noAvailableFiltersText: '',
+            select: 'multiple',
+            type: '',
+            values: [
+                {
+                    count: 9,
+                    id: '20',
+                    image: '',
+                    items: [],
+                    name: '22220',
+                    position: 'item',
+                    selected: false
+                },
+                {
+                    count: 1,
+                    id: '21',
+                    image: '',
+                    items: [],
+                    name: '22221',
+                    position: 'item',
+                    selected: false
+                }
+            ]
+        };
+
+        store.state.itemList = {
+            isLoading: false,
+            selectedFacets: [],
+            facets: [facet]
+        };
+
+        const wrapper = shallowMount(ItemDropdown, { propsData: { facet }, store, localVue });
+
+        expect(wrapper.element.querySelectorAll('.fl-dropdown-container').length).toBe(0);
+    });
+
+    it('shows one filter value inside a dropdown if there are two options and the fixed item count is set to 1', () => {
+        const facet: Facet = {
+            cssClass: '',
+            findologicFilterType: 'select',
+            id: 'test',
+            isMain: false,
+            itemCount: 1,
+            name: 'Test',
+            noAvailableFiltersText: '',
+            select: 'multiple',
+            type: '',
+            values: [
+                {
+                    count: 9,
+                    id: '20',
+                    items: [],
+                    name: '22220',
+                    position: 'item',
+                    selected: false
+                },
+                {
+                    count: 1,
+                    id: '21',
+                    items: [],
+                    name: '22221',
+                    position: 'item',
+                    selected: false
+                }
+            ]
+        };
+
+        store.state.itemList = {
+            isLoading: false,
+            selectedFacets: [],
+            facets: [facet]
+        };
+
+        const wrapper = shallowMount(ItemDropdown, { propsData: { facet }, store, localVue });
+
+        expect(wrapper.element.querySelectorAll(':scope > *').length).toBe(2);
+        expect(wrapper.element.querySelectorAll(':scope > div.form-check').length).toBe(1);
+        expect(wrapper.element.querySelectorAll(':scope > div.fl-dropdown-container.custom-select ul li').length).toBe(1);
```

Comment:
Additionally please also check the names of the filters. Having only these counts makes it a bit hard to see what this test is actually checking for.

Reply:
Filter names can't be checked here as they are outside of these components. That is done in `FindologicItemFilter.spec.ts`